tests/test_cli.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from ome_zarr_models.cli import main
+
+
+@pytest.mark.parametrize(
+    "url",
+    [
+        "https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.5/idr0066/ExpD_chicken_embryo_MIP.ome.zarr",
+        "https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.5/idr0062A/6001240_labels.zarr",
+        "https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.5/idr0010/76-45.ome.zarr",
+    ],
+)
+@pytest.mark.parametrize("cmd", ["validate", "info"])
+def test_cli(url: str, cmd: str, monkeypatch: pytest.MonkeyPatch) -> None:
```

Comment:
your call on whether you want to include this test.  can always be removed if those URLS become invalid (and currently the test will only xfail if the URL is invalid)